mmv1/templates/terraform/examples/cloudsecuritycompliance_framework_deployment_basic.tf.tmpl

```diff
@@ -0,0 +1,75 @@
+resource "google_cloud_security_compliance_framework" "{{$.PrimaryResourceId}}" {
+  organization = "{{index $.TestEnvVars "org_id"}}"
+  location     = "global"
+  framework_id = "{{index $.Vars "framework_name"}}"
+  
+  display_name = "Terraform Framework Name"
+  description  = "An Terraform description for the framework"
+  
+  cloud_control_details {
+		name              = "organizations/%{org_id}/locations/global/cloudControls/builtin-detective-policy-for-vertex-ai-runtime-template-idle-shutdown"
+		major_revision_id = "1"
+    
+    parameters {
+      name = "location"
+      parameter_value {
+        string_value = "us-central1"
+      }
+    }
+  }
+}
+
+resource "google_cloud_security_compliance_framework_deployment" "{{$.PrimaryResourceId}}" {
+  organization            = "{{index $.TestEnvVars "org_id"}}"
+  location                = "global"
+  framework_deployment_id = "{{index $.Vars "deployment_name"}}"
+  description             = "A framework deployment for cloud security compliance"
+  
+  framework {
+    framework         = google_cloud_security_compliance_framework.example.name
+    major_revision_id = "1"
+  }
+  
+  target_resource_config {
+    existing_target_resource = "organizations/{{index $.TestEnvVars "org_id"}}"
+  }
+  
+  cloud_control_metadata {
+    enforcement_mode = "DETECTIVE"
```

Comment:
Can you change the value for this field in these examples to also test `PREVENTIVE` and `AUDIT`?